docs/cvl/v5-changes.md

```diff
@@ -3,8 +3,9 @@ Certora CLI 5.0 Changes
 
 The release of `certora-cli` version 5.0 introduces a few small breaking
 changes for CVL.  These changes improve the coverage for parametric rules and
-invariants, and simplify some rarely-used features.  This document explains
-those changes and how to work with them.
+invariants, disallow Solidity function calls in quantified expressions,
+and simplify some rarely-used features.  This document explains those changes
+and how to work with them.
```

Comment:
I'm ok with this, although for this high-level summary I'm willing to consider the quantifier change to be included in "simplify some rarely-used features"

Reply:
I do not have a preference, so if someone wants to weigh in, now is the time.